tests/rados/pool_tests.py

```diff
@@ -361,3 +361,62 @@ def run(ceph_cluster, **kw):
             log.info(f"Completed the test of pg_min_num on pool: {entry['pool_name']} ")
         log.info("pg_min_num tests completed")
         return 0
+
+    if config.get("verify_osd_omap_entries"):
```

Comment:
Would it make sense to write a method containing the below workflow within this file/module?

Reply:
Added the workflow to a new file.